src/Microsoft.ML.PipelineInference/TransformInference.cs

```diff
@@ -92,6 +93,34 @@ public SuggestedTransform Clone()
             public override string ToString() => ExpertType.Name;
         }
 
+        public struct TransformString : IEquatable<TransformString>
+        {
+            public readonly string Kind;
+            public readonly string Settings;
+
+            public TransformString(string kind, string settings)
+            {
+                Kind = kind ?? "";
+                Settings = settings ?? "";
+            }
+
+            public bool Equals(TransformString other)
+            {
+                return Kind == other.Kind &&
+                    Settings.Equals(other.Settings);
```

Comment:
Can we please be consistent with equality comparison syntax?

Reply:
Oops - yes. Good catch. This was first taken from `SubComponent`, where `Settings` is an array.  Fixing.